### PR TITLE
Transcript Handling V2

### PR DESCRIPTION
**Pull request by alex1115alex**
Updated transcript handling for the (hopefully but probably not) final time.

Still needs a backslider => will be added soon enough... Realistically within 4.3 hours of you reading this text.

### CODE REVIEW

File: server/DatabaseHandler.py [hunk 1]
```diff
@@ -62,6 +63,20 @@ def initCacheCollection(self):
 
     ### MISC ###
 
+    # Returns the index of the nearest beginning of a word before "currIndex"
+    # EX: findClosestStartWordIndex('hello world, my name is alex!', 5) => 5
+    # EX: ...
+    # EX: findClosestStartWordIndex('hello world, my name is alex!', 11) => 5
+    # EX: findClosestStartWordIndex('hello world, my name is alex!', 12) => 12
+    def findClosestStartWordIndex(self, text, currIndex):
```
Comment (nic-olo) [inline server/DatabaseHandler.py:71]: nit: use snake_case

Comment (nic-olo) [reply server/DatabaseHandler.py:71]: same for the rest of the function

File: server/DatabaseHandler.py [hunk 2]
```diff
@@ -187,32 +232,26 @@ def getNewCseTranscriptsForUser(self, userId, deleteAfter=False):
         self.userCollection.update_one(filter=filter, update=update)
         return unconsumed_transcripts
 
+    def updateCseConsumedTranscriptIdxForUser(self, userId, newIndex):
```
Comment (nic-olo) [inline server/DatabaseHandler.py:235]: use snake_case

Comment (nic-olo) [reply server/DatabaseHandler.py:235]: same for the rest of the code

File: server/DatabaseHandler.py (hunk 1 above)
Comment (KenjiPcx) [reply server/DatabaseHandler.py:71]: nvm got what the function does now

File: server/DatabaseHandler.py [hunk 3]
```diff
@@ -62,6 +63,20 @@ def initCacheCollection(self):
 
     ### MISC ###
 
+    # Returns the index of the nearest beginning of a word before "currIndex"
```
Comment (nic-olo) [inline server/DatabaseHandler.py:66]: Docstring should be a triple-quoted string beneath the function name,
e.g.
def func_name(self, param):
"""
Docstring
"""

**Pull request merged**